views/login.rb

```diff
@@ -1,39 +1,35 @@
 module Omnigollum
   module Views
     class Login < Mustache
-      self.template_path = File.expand_path("../../templates", __FILE__)
+      self.template_path = File.expand_path('../templates', __dir__)
```

Comment:
What's the justification for this??

Reply:
Wrong PR. This is meant for the forked version.